lib/device-log/ios-crash-log.js

```diff
@@ -51,7 +51,7 @@ class IOSCrashLog {
       log.debug(`Crash reports root '${this.logDir}' does not exist. Got nothing to gather.`);
       return [];
     }
-    const foundFiles = await fs.glob(`${this.logDir}/**/*.crash`, {
+    const foundFiles = await fs.glob(`${this.logDir}/**/*.+(crash|ips)`, {
```

Comment:
Thank you, could you leave a comment that the `.ips` is since Monterey for a future note?

Reply:
Sure, added the comment. Thank you for taking a look.